### DISCUSSION TITLE
docs: rdoc link not working?

### DISCUSSION THREAD

**Discussion by benkoshy** [jeremyevans/sequel#1945] [General]
Steps: 
1. Click here: https://www.rubydoc.info/gems/sequel/3.15.0/Sequel%2FDataset:filter
2. Then click on the Virtual Rows link. https://www.rubydoc.info/files/doc/virtual_rows_rdoc.html

![image](https://user-images.githubusercontent.com/15097447/196804988-39639a54-ce3b-4c85-929e-0e46cb9af74d.png)

Nothing seems to come up? 

![image](https://user-images.githubusercontent.com/15097447/196805100-bd5745a5-4595-4e4d-b139-8707cd7615b3.png)

Same for the data set filtering guide link below it.

If this is an issue, not sure how to fix it.

**Comment by jeremyevans** [comment]
Sounds like a bug in rubydoc.info, or something long since fixed (Sequel 3.15.0 was released in September 2010).  Works fine on Sequel's current docs on Sequel's site: http://sequel.jeremyevans.net/rdoc/classes/Sequel/Dataset.html#method-i-where

If this is an issue with rubydoc.info with the current version of Sequel, you should probably report it to the rubydoc.info maintainers.
